core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -50,7 +60,8 @@ <h3>Upload Thumbnail</h3>
             <i class="material-icons oppia-vcenter">&#xE14C;</i>
           </button>
           <div>
-            <img ng-if="uploadedImage" class="oppia-thumbnail-image" ng-src="<[uploadedImage]>" height="<[aspectRatio === '4:3' ? '186px' : '180px']>" width="<[aspectRatio === '4:3' ? '248px' : '320px']>">
+            <thumbnail-display ng-if="uploadedImage" [classes]="['oppia-thumbnail-image']" [source]="uploadedImage" [height]="aspectRatio === '4:3' ? '186px' : '180px'" [width]="aspectRatio === '4:3' ? '248px' : '320px'"></thumbnail-display>
```

Comment:
The class name here is used to select the image element and update the background image color (not for styling purposes). Now that we have a `thumbnail-display` component, can we make it such that we just pass the background color to this component instead?